website/cue/reference/releases/0.46.0.cue

```diff
@@ -0,0 +1,361 @@
+package metadata
+
+releases: "0.46.0": {
+	date:     "2025-04-04"
+	codename: ""
+
+	whats_next: []
+
+	description: """
+		The Vector team is pleased to announce version `0.46.0`!
+
+		Release highlights:
+
+		- A new `postgres` sink is now available and it supports logs, metrics and traces!
```

Comment:
```suggestion
		- A new `postgres` sink is now available and it supports logs, metrics, and traces!
```

Reply:
I will revert this unrelated changes in next commit